typed-racket-lib/typed-racket/typecheck/tc-lambda-unit.rkt

```diff
@@ -269,8 +269,8 @@
                         body))])]))
 
 
-;; positional: natural? - the number of positional arguments
-;; rest: boolean? - if there is a positional argument
+;; positional: (listof identifier?)
+;; rest: id or #f
 ;; syntax: syntax? - the improper syntax list of identifiers
```

Comment:
Is the "improper syntax list" the same as the `(listof identifier?)` in the positional field?

(The comment makes it sound like it is. I think `syntax` must be something different, but not sure what that could be.)

Reply:
yes and no... its more or less `(append positional (or id '()))` -- i.e. what `read` produces when reading the function's definition (right?).